src/status_im/contexts/wallet/create_account/edit_derivation_path/component_spec.cljs

```diff
@@ -1,36 +0,0 @@
-(ns status-im.contexts.wallet.create-account.edit-derivation-path.component-spec
```

Comment:
it would be nice if you provided a reason for deleting this test file 👍 
can you explain why you did this?

Reply:
@J-Son89 it wasn't testing anything useful. It was based on the initial implementation of the screen which was incorrectly made for the [Keycard derivation path flow](https://www.figma.com/file/HKncH4wnDwZDAhc4AryK8Y/Wallet-for-Mobile?type=design&node-id=2228-489095&mode=design) (I think an external candidate worked on it)

Also, a general point, are we writing component tests for whole screens? Cause I don't think that is very helpful